packages/kubernetes-1.16/kubelet-config

```diff
@@ -68,6 +68,12 @@ kubeReserved:
   {{~/if}}
   {{~/if}}
   ephemeral-storage: "{{default "1Gi" settings.kubernetes.kube-reserved.ephemeral-storage}}"
+{{~#if settings.kubernetes.system-reserved}}
```

Comment:
I'm not sure if this is already covered. Can you launch an instance with none of these set to ensure that this `if` block works as expected? I think this is the first time we have used it on a map/object type.

Reply:
Yeah. I had tested that launch an instance with none of these set, and `if` did block. I tried all combinations like only set `cpu` or only set `cpu` and `memory` to test if map/object type worked here. All of them worked as expected. : )